rpush.gemspec

```diff
@@ -42,7 +42,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rubocop'
   s.add_development_dependency 'byebug'
 
-  s.add_development_dependency 'pg'
+  s.add_development_dependency 'pg', '~> 0.21'
```

Comment:
I don't think we should limit the `pg` gem here. Maybe within the Appraisals, Rails 5.1.5 and 5-stable seem to have support for pg 1.0. https://github.com/rails/rails/pull/31671#issuecomment-357605227